examples/07-routing/src/app.gleam

```diff
@@ -158,16 +141,41 @@ fn render_welcome(model: Model, slug) -> Element(a) {
     |> list.find(fn(guest: Guest) { guest.slug == slug })
 
   let title = case guest {
-    Ok(guest) -> to_title("Hello, " <> guest.name <> "! 🎉")
-    _ -> to_title("Sorry ... didn't quite catch that.")
+    Ok(guest) -> render_title("Hello, " <> guest.name <> "! 🎉")
+    _ -> render_title("Sorry ... didn't quite catch that.")
+  }
+
+  render_body([title])
+}
+
+fn render_nav(model: Model) -> Element(a) {
+  let item_styles = [#("margin", "1rem"), #("text-decoration", "underline")]
+
+  let render_nav_item = fn(path, text) {
+    html.a([attribute.href("/" <> path), attribute.style(item_styles)], [
+      element.text(text),
+    ])
   }
 
-  ui.centre([attribute.style([#("margin-top", "10vh")])], title)
+  let guest_nav_items =
+    model.guests
+    |> list.map(fn(guest: Guest) {
+      render_nav_item("welcome/" <> guest.slug, guest.name)
+    })
+
+  html.nav([], [render_nav_item("", "Home"), ..guest_nav_items])
```

Comment:
Instead of using inline styles to add margin, lets do `cluster.of(ui.nav, [], [...])`

Reply:
I _think_ I got this one but the actual syntax that worked for me was:

`  ui.cluster([], [html.nav([], [view_nav_item("", "Home"), ..guest_nav_items])])
`

Reply:
no no we want:
```
import lustre/ui/layout/cluster

...

cluster.of(html.nav, [], [ view_nav_item(...), ... ])`
```
this is a feature of lustre/ui!

Reply:
Ohhhhhh. Got it!